build.gradle

```diff
@@ -21,11 +21,12 @@ repositories {
 dependencies {
     compile group: 'commons-io', name: 'commons-io', version: '2.6'
     compile group: 'net.lingala.zip4j', name: 'zip4j', version: '2.3.0'
-    compile group: 'org.scala-lang', name: 'scala-library', version: '2.12.10'
+    compileOnly group: 'org.scala-lang', name: 'scala-library', version: '2.12.10'
```

Comment:
I'm a bit skeptical when the configs have to be changed due to a "does not work on my machine"

Reply:
There is definitely an issue if Scala libraries are loaded both by our plugin and Scala plugin. The dubious part of the original config is `compile` keyword which is deprecated in Gradle in favor of keywords `api`/`implementation`/`compileOnly` whose behavior is more easily understandable and straightforward. I guess that in some setups, `compile` works here but causes troubles in others. Instead of trying to understand the quirks of `compile`, it is better to replace it with a more specific keyword.